superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -325,6 +325,11 @@ export default class FilterableTable extends PureComponent<
     return (a: Datum, b: Datum) => {
       const aValue = a[sortBy];
       const bValue = b[sortBy];
+
+      // Parse any floating numbers so they'll sort correctly
+      const parseFloatingNums = (value: any) =>
+        value.isNan ? value : parseFloat(value);
```

Comment:
that's cool, didn't know that isNan was a function!

Reply:
Oops, it's `Number.isNan` 😅